Megaparsec/Errors/ParseError.lean

```diff
@@ -82,11 +82,11 @@ def parseErrorTextPretty : ParseError β E → String
       else
         let o := Option.map (fun ei => [toString ei]) us
         messageItemsPretty "unexpected " (o.getD []) ++
-        messageItemsPretty "expecting " (List.map toString es.keys.toList)
+        messageItemsPretty "expecting " (List.map toString es.keys.toArray.toList)
```

Comment:
I think there's a direct `toList`, from what I recall

Reply:
Oh you're right, but for some reason the dot notation doesn't work for it...

Reply:
`        messageItemsPretty "expecting " (List.map toString (RBMap.Keys.toList es.keys))`

Reply:
Alright. Is `.data` faster than `.toList`?